src/client/datascience/jupyter/kernels/kernelSelector.ts

```diff
@@ -233,6 +233,7 @@ export class KernelSelector implements IKernelSelectionUsage {
     // eslint-disable-next-line complexity
     @traceDecorators.info('Get preferred remote kernel connection')
     @reportAction(ReportableAction.KernelsGetKernelForRemoteConnection)
+    @captureTelemetry(Telemetry.GetPreferredKernelPerf)
```

Comment:
We didn't have this telemetry, re-used existing one, we can didfferentiate whether its local or remote based on shared properties.